components/script/dom/bindings/reflector.rs

```diff
@@ -141,6 +141,7 @@ impl MutDomObject for Reflector {
 }
 
 /// A trait to provide a function pointer to wrap function for DOM objects.
+#[allow(clippy::type_complexity)]
```

Comment:
that would be in `components/script/dom/bindings/iterable.rs`, right?

Reply:
I mean to move this down to the constant defined inside the trait instead of on the entire trait. For example in this case it would move to right before line 147.